test/common/README.md

```diff
@@ -261,6 +261,124 @@ Platform normalizes the `pwd` command.
 
 Synchronous version of `spawnPwd`.
 
+### tagGlue()
+* return [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type)
+
+Tag function for tagged template literals:
+removes all literal line breaks and leading spaces.
+
+<!-- eslint-disable strict -->
+```js
+const assert = require('assert');
+const common = require('../common');
+const { tagGlue } = common;
+
+const unTagged =
+               '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+
+const tagged = tagGlue`
+  0123456789
+  ABCDEFGHIJKLMNOPQRSTUVWXYZ
+  abcdefghijklmnopqrstuvwxyz
+`;
+
+assert.strictEqual(unTagged, tagged);
+```
+
+### tagUnwrap()
```

Comment:
It seems pretty usual term in text editors (also used in our style guide) for visual formatting. Your variants seem to suppose more aggressive editing.